public/store/css/main.css

```diff
@@ -589,6 +593,7 @@ EDIT PAGE
 
 .clickable-span {
 	cursor: pointer;
+  text-decoration: underline;
```

Comment:
lol gross... just kidding - but you need couple more spaces to get the indent matching.